kits/js/README.md

```diff
@@ -0,0 +1,36 @@
+# Lux AI Season 3 JS Kit
+
+This is the folder for the JS kit. Please make sure to read the instructions as they are important regarding how you will write a bot and submit it to the competition.
+
+Make sure to check our [Discord](https://discord.gg/aWJt3UAcgn) for announcements if there are any breaking changes.
+<!--  or the [Kaggle forums](https://www.kaggle.com/c/lux-ai-season2/discussion)  -->
+## Requirements
+
+You will need Node 18 or higher installed.
```

Comment:
can you add an comment about where to look to install node js? I usually point users to nvm